patchwork/app.py

```diff
@@ -232,6 +291,22 @@ def cli(
             logger.debug(traceback.format_exc())
             logger.error(f"Error running patchflow {patchflow}: {e}")
             exit(1)
+    else:
+        with logger.panel(f"Patchflow {patchflow} logs") as _:
+            if inputs.get("debug") is not None:
```

Comment:
If the debug flag is not set there is no need to handle the debug value on this else block.